documentation/src/main/asciidoc/user_guide/chapter-78-Scripting.adoc

```diff
@@ -41,6 +41,13 @@ The following metadata property keys are available
 * collator: this property is only valid for +mapper+ scripts, and specifies the name of another script that will be used for the collation phase of a map/reduce job. This property will be removed in Infinispan 9.
 * combiner: this property is only valid for +mapper+ scripts, and specifies the name of another script that will be used for the combine phase of a map/reduce job. This property will be removed in Infinispan 9.
 * parameters: an array of valid parameter names for this script. Invocations which specify parameter names not included in this list will cause an exception.
+* datatype: optional property providing information, in the form of
```

Comment:
@rigazilla you want to take note and implement support for this in the C++ client too (simpler than the JBasicMarshaller)